templates/validation-environment-template.js

```diff
@@ -1,19 +1,19 @@
 function makeValidationEnvironment(_, simple) {
-  var doc = { };
```

Comment:
Since the sync function template's `.jshintrc` was moved into that subdirectory, this file is free to use ECMAScript 2015 features like `const`.